pyface/data_view/exporters/item_exporter.py

```diff
@@ -0,0 +1,47 @@
+
+from pyface.data_view.abstract_data_exporter import AbstractDataExporter
+from pyface.data_view.data_view_errors import DataViewGetError
+
+
+class ItemExporter(AbstractDataExporter):
+    """ Export a single item from a data view.
+
+    This is suitable for drag and drop or copying of the content of a single
+    item in a data view.  If passed an multiple items it will fail by
+    raising ``DataViewGetError``; drag and drop support will then ignore this
+    as an exporter to use.
+    """
+
+    def add_data(self, data_wrapper, model, indices):
+        """ Add data to the data wrapper from the model and indices.
+
+        Parameters
+        ----------
+        data_wrapper : DataWrapper instance
+            The data wrapper that will be used to export data.
+        model : AbstractDataModel instance
+            The data model holding the data.
+        indices : list of (row, column) index pairs
+            The indices where the data is to be stored.
+        """
+        # only export single item values
+        if len(indices) == 1:
+            super().add_data(data_wrapper, model, indices)
+
+    def get_data(self, model, indices):
+        """ Get the data to be exported from the model and indices.
+
+        Parameters
+        ----------
+        model : AbstractDataModel instance
+            The data model holding the data.
+        indices : list of (row, column) index pairs
+            The indices where the data is to be stored.
+
+        Returns
+        -------
+        data : any
+            The data, of a type that can be serialized by the format.
+        """
+        row, column = indices[0]
+        return self.get_value(model, row, column)
```

Comment:
i'm not sure how this is going to raise a `DataViewGetError` as mentioned in the class docstring.